bindings/py/dummy.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+int main()
+{
+    return 0;
+}
+
+void initdummy() {}
```

Comment:
Make sure you have a newline at end of file. Normally this is just a stylistic request but it can actually have an impact with C++ code.